### PR TITLE
Expose the checks used by the check container and check operator actions in their help outputs

### LINKED ISSUE #43: Expose the checks used by the `check container` and `check operator` actions in their help outputs
It may be useful for the user to know exactly what tests are going to be executed when they call container/operator check commands.

### PR DESCRIPTION
**Pull request by rocrisp**
Expose the checks used by the check container and check operator actions in their help outputs
fixes #43

### CODE REVIEW

**Comment by rocrisp** [comment]
Example of the output:

$ ./preflight check operator -h
This command will run the Certification checks for an Operator bundle image.

The checks that will be executed are the following:
     - ValidateOperatorBundle
     - ScorecardBasicSpecCheck
     - ScorecardOlmSuiteCheck
     - OperatorImageSourceRegistryCheck

Usage:
  preflight check operator <url to Operator bundle image> [flags]
	
Flags:
  -h, --help   help for operator
------------

$ ./preflight check container -h
This command will run the Certification checks for a container image.

The checks that will be executed are the following:
     - HasRequiredLabel
     - BasedOnUbi
     - HasLicense
     - HasMinimalVulnerabilities
     - HasUniqueTag
     - HasNoProhibitedPackages
     - RunAsNonRoot
     - LayerCountAcceptable

Usage:
  preflight check container <url to container image> [flags]
	
Flags:
  -h, --help   help for container

**Review by samira-barouti — CHANGES_REQUESTED** [review]
I think there are commits from your other PR is here. Please make sure to push only the relevant changes.

File: cmd/check_container.go [hunk 1]
```diff
@@ -81,12 +81,22 @@ var checkContainerCmd = &cobra.Command{
 }
 
 func init() {
-	checkContainerCmd.SetUsageTemplate(
-		`Usage:
+	checks := engine.ContainerPolicy()
+	checkString := "\n"
+	for _, value := range checks {
```
Comment (samira-barouti) [inline cmd/check_container.go:86]: you can replace this for loop with `strings.Join(engine.ContainerPolicy(), '-')`.

**Review by samira-barouti — CHANGES_REQUESTED** [review]
This is a personal opinion/preference: I am a bit on fence with using --help to display the list of checks. As an end user, I expect --help show me how to use the tool. I think sth like --show-all or --show might be another alternative.

File: cmd/check_container.go (hunk 1 above)
Comment (rocrisp) [reply cmd/check_container.go:86]: That's cleaner. 
suggestion for adding a \n after each value?

Comment (rocrisp) [reply cmd/check_container.go:86]: nvm, I added \n and that did the trick

**Review by jomkz — APPROVED** [review]
/lgtm

**Review by samira-barouti — APPROVED** [review]
/lgtm

**Comment by rocrisp** [comment]
I don't have write access to this repository so someone else needs to merge this for me, please

**Pull request merged**